src/ethereum_test/fill.py

```diff
@@ -46,14 +46,20 @@ def fill_test(
                 head=head,
                 fork=fork,
                 pre_state=test.pre,
-                post_state=None,
+                post_state=alloc,
                 seal_engine=engine,
+                name=test.name,
+                index=index + 1,
```

Comment:
Is this to not 0 index them?

Reply:
Yeah, I don't know why I did that, but I set it to zero now :+1: